.github/workflows/setup-branch-protection.yml

```
@@ -0,0 +1,61 @@
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/setup-branch-protection.yml'
+
+permissions:
+  contents: read
+  administration: write
```

actionlint (1.7.7)

unknown permission scope "administration". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)